manifest.md

```diff
@@ -1,36 +1,22 @@
 Manifest.json fields
 ====================
 
-## Standard fields
-
-(In alphabetical order.)
-
-* Field `display`: http://www.w3.org/TR/appmanifest/#display-member
-
-* Field `icons`: http://www.w3.org/TR/appmanifest/#icons-member
-
-Basic implementation only.
-
-* Field `name`: http://www.w3.org/TR/appmanifest/#name-member
-
-* Field `short_name`: http://www.w3.org/TR/appmanifest/#short_name-member
-
-* Field `start_url`: http://www.w3.org/TR/appmanifest/#start_url-member
-
-## Extension fields
-
-(In alphabetical order.)
-
-* Extension field `xwalk_app_version`: Version number, between 1 and 99.99.999
-
-* Extension field `xwalk_package_id`: Unique package identifier, e.g. `com.example.foo`.
-
-* Extension field `xwalk_target_platforms`: Target platform, e.g. `android`. At the moment only a single value is supported.
-
-* Extension field `xwalk_android_animatable_view`: Whether to allow zooming the browser view (`true`/`false`).
-
-* Extension field `xwalk_android_keep_screen_on`: Whether to keep the screen on while the app is in front (`true`/`false`).
-
-* Extension field `xwalk_windows_update_id`: Automatically generated identifier for native windows support (work in progress).
-
-* Extension field `xwalk_windows_vendor`: Vendor string for native windows support (work in progress).
+### Mandatory fields
+
+* `name`:	http://w3c.github.io/manifest/ Will be uses as label in the android launcher.
+* `icons`:	http://w3c.github.io/manifest/ Basic support, only one resolution per icon entry.
+* `start_url`:	http://w3c.github.io/manifest/ First html page to load.xw
+* `xwalk_app_version: "a.b.c"`: Three part application version number e.g. 1.2.3 needed for android and windows packaging.
+* `xwalk_package_id: "com.example.foo"`: Canonical name e.g. com.example.foo.
+
+### Extra fields
+
+* `display`:	http://w3c.github.io/manifest/ Values "standalone" and "fullscreen" are supported on Android.
```

Comment:
I believe they are the only fields supported on any platform (at least Windows), make it general?